content/en/blog/2021/istio-4th-birthday/index.md

```diff
@@ -0,0 +1,65 @@
+description: "Celebrating Istio’s 4th birthday."
+publishdate: 2021-05-24
+attribution: "Maria Cruz (Google), Aizhamal Nurmamat kyzy (Google)"
+keywords: [community,birthday,celebration]
+---
+# Celebrating Istio’s 4th birthday
+
+Four years ago today, the Istio project was born to the open source world. To celebrate this anniversary,
+we are hosting a week-long birthday celebration  that focuses on contributions to the Istio project that
+stem from using Istio in production. Read on to learn how to participate in this celebration  and enter a
+chance to win some Istio swag.
+
+{{< image link="./1.png" caption="Istio's 4th Birthday!" >}}
+
+## A year of important developments for Istio
+
+Over the last 12 months, the Istio project has been very focused on the [day-0
+& day-1](https://dzone.com/articles/defining-day-2-operations) experience for
+users by actively listening to our users through UX surveys and GitHub issues.
+
+- We [simplified the control plane architecture](https://istio.io/latest/blog/2020/istiod/) and
+   made Istio easier to install, configure and upgrade.
+- We provided clarity and process to our feature status and promotion of features and APIs.
+- We simplified debugging experience with various istioctl commands.
+- We expanded the mesh to services running in [VMs](https://istio.io/latest/news/releases/1.9.x/announcing-1.9/#virtual-machine-integration-beta)
+    and [multiclusters](https://istio.io/latest/docs/setup/install/multicluster/).
+- We made [statefulset easier](https://istio.io/latest/blog/2021/statefulsets-made-easier/) to use in Istio 1.10 with zero-configuration.
+- We made various performance improvements to Istio control plane and data plane via [discovery selectors](https://istio.io/latest/blog/2021/discovery-selectors/),
+    sidecar resources etc.
+- We [introduced WebAssembly](https://istio.io/latest/blog/2021/wasm-progress/) as our extensibility platform which has helped users tailor Istio to their needs
+- We beefed up our CVE management and release processes to meet enterprise needs.
+
+[Read more](https://istio.io/latest/blog/2020/tradewinds-2020/) about improvements to
+Istio in 2020 that made this technology easier to use.
+
+In February 2021, we celebrated the first IstioCon! This community-led event was an opportunity
+for users and developers to share [many examples](https://www.youtube.com/playlist?list=PL7wB27eZmdffS-g_xh7X-b0echc_XZMKV)
+of how they use Istio in production and lessons
+learned from it. IstioCon was a great opportunity to feature more than 25 end-user companies,
+like Salesforce, T-Mobile, and Airbnb, among others, to feature maintainers from across the Istio
+ecosystem, and to share the [Istio project roadmap](https://www.youtube.com/watch?v=WmjTeN-jtdY).
+
+This inaugural community conference was a major success, with more than 4,000 registrants from 80
+countries that participated in this conference. The program was conducted on US and Asia time zones,
```

Comment:
Doesn't sound correct. Maybe: s/that participated in this conference/participating

Reply:
fixed